c++/src/kj/refcount.h

```diff
@@ -134,10 +134,17 @@ class AtomicRefcounted: private kj::Disposer {
   KJ_DISALLOW_COPY(AtomicRefcounted);
 
   inline bool isShared() const {
+    return getRefcount() > 1;
+  }
+
+  inline uint getRefcount() const {
+    // This value can be useful for heuristic purposes, such as estimating the load on a resource.
+
 #if _MSC_VER
-    return KJ_MSVC_INTERLOCKED(Or, acq)(&refcount, 0) > 1;
+    // A static_cast is fine here because sizeof(uint) == sizeof(long) on MSVC.
```

Comment:
I suppose if you wanted to explicitly enforce this, you could throw in a `static_assert(...)`.

Reply:
True, I'll do that instead of the `static_cast`.